pkgs/applications/emulators/wine/base.nix

```diff
@@ -107,7 +107,7 @@ stdenv.mkDerivation ((lib.optionalAttrs (buildScript != null) {
      libX11 libXi libXcursor libXrandr libXrender libXxf86vm libXcomposite libXext
   ])
   ++ lib.optionals waylandSupport (with pkgs; [
-     wayland libxkbcommon wayland-protocols wayland.dev libxkbcommon.dev
+     wayland libxkbcommon wayland-protocols wayland.dev libxkbcommon.dev mesa.dev
```

Comment:
Why is this required?

Reply:
[This header](https://gitlab.collabora.com/alf/wine/-/blob/wayland/dlls/winewayland.drv/waylanddrv.h#L28) includes `<gbm.h>`; I couldn't find a libgbm in nixpkgs, and mesa is the only place in my Nix store that I found a `gbm.h` file. :shrug: Is there a better dependency for me to take?

Reply:
No that's alright then.

Reply:
Maybe you should put a comment on it.
Also, a bit OT, searching though nixpkgs there are other packages that use `mesa` just for `gbm` (I found at least 15 of them thanks to comments), however they simply use `mesa` in `buildInputs` instead of `mesa.dev`. Should that be changed?

Reply:
BuildInputs is magic and automatically chooses the dev input I believe.

We should definitely try and decouple gbm.h from mesa to reduce rebuilds when we try to update the driver.